ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -356,10 +356,10 @@ objects:
           name: ${CLUSTER_NAME}
         platform:
           openstack:
+            apiFloatingIP:    ${LB_FIP}
             cloud:            ${OS_CLOUD}
-            externalNetwork:  ${OPENSTACK_EXTERNAL_NETWORK}
             computeFlavor:    ${OPENSTACK_FLAVOR}
-            lbFloatingIP:     ${LB_FIP}
+            externalNetwork:  ${OPENSTACK_EXTERNAL_NETWORK}
```

Comment:
? not important but it's usually better to make the smallest amount of changes per commit. This  is necessary.